alphagov/collections
# Display timeline by nation on coronavirus landing page

Trello cards:
- https://trello.com/c/Bn4ekYnO
- https://trello.com/c/eE62ythU

Supersedes: #2437

Paired with: @andysellick 

# What's changed?

Adds a nation picker to the coronavirus landing page to show timeline information.

With JS disabled, this works using a standard form GET submission to show/hide the relevant section of the page. If JS is enabled, we hide the form submit button and clicking a radio button automatically switches the visibility of the relevant section.

Includes aria-live attributes so that changing the radio button announces the results of the change, and tracking attributes on the radio buttons themselves to give us some stats about usage (if cookies have been consented to).

# Why?

So that users can access specific information relevant to their location.

# Expected changes

## With timeline entries

<img width="1648" alt="Screenshot 2021-07-01 at 17 07 12" src="https://user-images.githubusercontent.com/5793815/124156363-756a2080-da8f-11eb-84c6-b8402a97045c.png">

## When there are no changes

<img width="1628" alt="Screenshot 2021-07-01 at 16 09 38" src="https://user-images.githubusercontent.com/5793815/124156773-f1fcff00-da8f-11eb-8bb6-f25ef876b3ce.png">

# How to test locally

The changes can only be seen in a development environment.

To see the changes pass `nation` as a querystring parameter.

To see England as the default: http://localhost:3070/coronavirus?nation=foo
To see another nation selected: http://localhost:3070/coronavirus?nation=wales
To see the current live content item (without nations): http://localhost:3070/coronavirus




⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️
